File: frontend/src/types/AccountTrendDto.ts
import SubcategoryOut from "@/src/types/SubcategoryOut";
import CategorySummary from "@/src/types/CategorySummary";

export default class AccountTrendDto{
    public constructor(
        public year: number | null,
        public month: number | null,
        public balance: string,
        ) {}
}

File: app/app/src/main/java/com/xavierclavel/expenses_tracker/model/AccountTrendDto.kt
package com.xavierclavel.expenses_tracker.model

import kotlinx.serialization.Serializable

@Serializable
data class AccountTrendDto(
    val year: Int,
    val month: Int? = null,
    val balance: String,
    val change: String? = null,
    val proportionalChange: String? = null,
)
